cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ShowProceduresTest.scala

```diff
@@ -0,0 +1,88 @@
+ */
+package org.neo4j.cypher.docgen
+
+import org.neo4j.cypher.docgen.tooling._
+
+class ShowProceduresTest extends DocumentingTest {
+
+  override def outputPath = "target/docs/dev/ql/listing"
+
+  override def doc: Document = new DocBuilder {
+    doc("Listing procedures", "query-listing-procedures")
+    p("Listing the available procedures can be done with `SHOW PROCEDURES`.")
+    p("include::listing-procedures-syntax.asciidoc[]")
+    p("This command will produce a table with the following columns:")
+    p("include::listing-procedures-columns.asciidoc[]")
+    section("Examples") {
+      section("Listing all procedures") {
+        p(
+          """To list all available procedures with the default output columns, the `SHOW PROCEDURES` command can be used.
+            |If all columns are wanted, use `SHOW PROCEDURES YIELD *`.""".stripMargin)
+        query("SHOW PROCEDURES", ResultAssertions(p => {
+          p.columns should contain theSameElementsAs Array("name", "description", "mode", "worksOnSystem")
+        })) {
+          resultTable()
+        }
+      }
+      section("Listing procedures with filtering on output columns") {
+        p(
+          """The listed procedures can be filtered in multiple ways, one way is to use the `WHERE` clause.
+            |For example getting the name of all admin procedures:""".stripMargin)
+        query("SHOW PROCEDURES YIELD name, admin WHERE admin", ResultAssertions(p => {
+          p.columns should contain theSameElementsAs Array("name", "admin")
+          p.columnAs[Boolean]("admin").foreach(_ should be(true))
+        })) {
+          resultTable()
+        }
+      }
+      section("Listing procedures with other filtering") {
+        database("system")
+        initQueries(
+          "CREATE USER jake SET PASSWORD 'abc123' CHANGE NOT REQUIRED"
+        )
+        database("neo4j")
+        p(
+          """The listed procedures can also be filtered on whether a user can execute them.
+            |This filtering is only available through the `EXECUTABLE` clause and not through the `WHERE` clause.
+            |This is due to using the users privileges instead of just filtering on the available output columns.""".stripMargin)
+        p(
+          """There are two versions of the `EXECUTABLE` clause.
+            |The first one is to filter for the current user:""".stripMargin)
+        login("jake", "abc123")
+        query("SHOW PROCEDURES EXECUTABLE BY CURRENT USER YIELD *", ResultAssertions(p => {
+          p.columns should contain theSameElementsAs Array("name", "description", "mode", "worksOnSystem", "signature", "argumentDescription", "returnDescription", "admin", "rolesExecution", "rolesBoostedExecution", "option")
+          p.columnAs[List[String]]("rolesExecution").foreach(_ should be(null))
+          p.columnAs[List[String]]("rolesBoostedExecution").foreach(_ should be(null))
+        })) {
+          resultTable()
```

Comment:
Again, many narrow columns are hard to read and 57 rows is a lot:
![image](https://user-images.githubusercontent.com/1756429/117289114-22dfff80-ae64-11eb-97d4-745a3eab613a.png)

Reply:
I know, all of these are horrible result tables.

But I don't want to get rid of all tables and I want to have different types of queries and not just have yield and return on all of them so I'm not sure how to fix this...

I could maybe try to make some custom resultTable methods that just prints the first 10 rows and reorder the columns to the order we want. can't really do anything about the amount of columns when using `yield *` though

Reply:
I think a shorter result table would be generally useful, so yeah that sounds good.